spec/higher_level_api/integration/connection_spec.rb

```diff
@@ -319,9 +343,9 @@
           password: "bunny_password",
           vhost: "bunny_testbed",
           ssl:                 true,
-          ssl_cert:            "spec/tls/client_cert.pem",
```

Comment:
`:ssl` context test data setup is outdated, so I normalized it with `:tls` context,